saritasa_invocations/pip.py

```diff
@@ -15,7 +15,20 @@ def install_dependencies(
 
 
 @invoke.task
```

Comment:
Add this https://peps.python.org/pep-0702/#example

Reply:
Exists only for python3.13 as I can understand

<img width="926" alt="image" src="https://github.com/saritasa-nest/saritasa-invocations/assets/30920379/d75c7197-aaf0-42bd-84c8-03d36cf14e35">

<img width="1066" alt="image" src="https://github.com/saritasa-nest/saritasa-invocations/assets/30920379/7e5ec8de-dd8b-481f-bd26-1e8b8d94a219">